Code/Core/Process/SystemMutex.cpp

```diff
@@ -58,15 +58,16 @@ bool SystemMutex::TryLock()
 		return true;
     #elif defined( __LINUX__ ) || defined( __APPLE__ )
         AStackString<> tempFileName;
-        tempFileName.Format( "/var/run/%s.lock", m_Name.Get() );
+        tempFileName.Format( "/tmp/%s.lock", m_Name.Get());
         m_Handle = open( tempFileName.Get(), O_CREAT | O_RDWR, 0666 );
         int rc = flock( m_Handle, LOCK_EX | LOCK_NB );
         if ( rc )
         {
-            if ( errno == EWOULDBLOCK )
+            if ( errno == EAGAIN )
```

Comment:
When I wrote this code, I verified that the return value was EWOULDBLOCK. Did you get EAGAIN when running a second FASTBuild? Should we be checking for both?